resources/documentation/learn_ansible.rst

```diff
@@ -5,26 +5,25 @@ Learn Ansible
 To understand how **BlueBanquise** works, it is essential to learn basis of
 Ansible.
 
-This section tries to provide key knowledge to at least be able to manipulate
+This section provides key knowledge to, at least, be able to manipulate
 and edit the stack to your needs.
 
-It is assumed here that your system is already installed, and that Ansible has
-also been installed. It is also assumed that **BlueBanquise** stack has not been
-installed yet, and so Ansible is freshly installed without modifications.
+It is assumed that your system and Ansible are installed. It is also assumed that the **BlueBanquise** stack is NOT yet
```

Comment:
Can you keep the 80 columns layout? It helps to read in the terminal.

Reply:
Of course @btravouillon